bin/i18n/sync-hourofcode-out

```diff
@@ -90,7 +90,7 @@ def clean_crowdin_markdown_errors
   # fix social media buttons in promote/index
   Dir.glob("pegasus/sites.v3/hourofcode.com/i18n/public/**/promote/index.md").each do |file|
     puts file
-    string_replacement = '#{request.host}'
+    string_replacement = '#{request.host}'  # rubocop:disable Lint/InterpolationCheck
```

Comment:
Prefer `request.host.to_s`?

Reply:
Per my earlier comment, the `string_replacement` is being set to the literal string `#{request.host}`, there is no interpolation happening on this line. Doing `request.host.to_s` would perform that interpolation.